sonarqube/README.md

```diff
@@ -0,0 +1,188 @@
+# Agent Check: SonarQube
+
+## Overview
+
+This check monitors [SonarQube][1].
+
+## Setup
+
+### Installation
+
+The SonarQube check is included in the [Datadog Agent][2] package.
+No additional installation is needed on your server.
+
+### Configuration
+
+<!-- xxx tabs xxx -->
+<!-- xxx tab "Host" xxx -->
+
+#### Host
+
+To configure this check for an Agent running on a host:
+
+##### Metric collection
+
+1. Edit the `sonarqube.d/conf.yaml` file, in the `conf.d/` folder at the root of your
+   Agent's configuration directory to start collecting your SonarQube data.
+   See the [sample sonarqube.d/conf.yaml][3] for all available configuration options.
```

Comment:
Links within tabs must be numbered and referenced within the `xxx tab "Name" xxx` tags. So this link should be `[1]` and the url should appear on line 89 and forward. Similar with other links on this tab and on the "Containerized" tab.

Reply:
actually false :) Withdrawing comment